src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -509,6 +509,16 @@ private function addSessionSection(ArrayNodeDefinition $rootNode)
                             ->defaultValue(0)
                             ->info('seconds to wait between 2 session metadata updates')
                         ->end()
+                        ->integerNode('sid_length')
+                            ->defaultValue(32)
+                            ->min(22)
+                            ->max(256)
```

Comment:
please add `->info()` node which explains what it does (and maybe why it could be useful?)

Reply:
Shoud I really add Info about php ini session option ? 
Based on the others options, like `cookies_only`, that doesn't have the `->info()` too...

Reply:
It would be a nice addition. If anything it can mention the php ini setting names and refer to those.

Reply:
there names itself are the php init setting names...
Like others : `cookie_httponly`, `cookie_samesite`, etc... etc...

And the mention to php ini settings are done in the Framework documentation https://symfony.com/doc/current/reference/configuration/framework.html#session
(version 4.3 pull request in progress)

Therefore I mention it, in the changelog for the 4.3 version

Reply:
I'm fine not adding the info personal: why would we add it for these rarely used options and not for the main ones? if we want to add them to all options, maybe, but in another PR :)